operations/deployment/crunchy-postgres/templates/PostgresCluster.yaml

```
@@ -0,0 +1,188 @@
+        resources:
+          requests:
+            cpu: {{ .Values.pgmonitor.exporter.requests.cpu }}
+            memory: {{ .Values.pgmonitor.exporter.requests.memory }}
+          limits:
+            cpu: {{ .Values.pgmonitor.exporter.limits.cpu }}
+            memory: {{ .Values.pgmonitor.exporter.limits.memory }}
+
+  {{ end }}
+
+  # This requires PG0 v5.1.0
+  # https://github.com/CrunchyData/postgres-operator/issues/3032
+  {{ if .Values.userInterface.enabled }}
+
+  userInterface:
+    pgAdmin:
+      image: {{ .Values.userInterface.pgAdmin.image }}
+      dataVolumeClaimSpec:
+        accessModes:
+        - {{ .Values.userInterface.pgAdmin.dataVolumeClaimSpec.accessModes }}
+        resources:
+          requests:
+            storage: {{ .Values.userInterface.pgAdmin.dataVolumeClaimSpec.requests.storage }}
+        
+  {{ end }}
+
+  instances:
+    - name: {{ .Values.instances.name }}
+      replicas: {{ .Values.instances.replicas }}
+      resources:
+        requests:
+          cpu: {{ .Values.instances.requests.cpu }}
+          memory: {{ .Values.instances.requests.memory }}
+        limits:
+          cpu: {{ .Values.instances.limits.cpu }}
+          memory: {{ .Values.instances.limits.memory }}
+      sidecars:
+        replicaCertCopy:
+          resources:
+            requests:
+              cpu: {{ .Values.instances.sidecars.replicaCertCopy.requests.cpu }}
+              memory: {{ .Values.instances.sidecars.replicaCertCopy.requests.memory }}
+            limits:
+              cpu: {{ .Values.instances.sidecars.replicaCertCopy.limits.cpu }}
+              memory: {{ .Values.instances.sidecars.replicaCertCopy.limits.memory }}
+      dataVolumeClaimSpec:
+        accessModes:
+          - "ReadWriteOnce"
+        resources:
+          requests:
+            storage: {{ .Values.dataVolumeClaimSpec.storage }}
+        storageClassName: netapp-block-standard
+      affinity:
+        podAntiAffinity:
+          preferredDuringSchedulingIgnoredDuringExecution:
+            - weight: 1
+              podAffinityTerm:
+                topologyKey: kubernetes.io/hostname
+                labelSelector:
+                  matchLabels:
+                    postgres-operator.crunchydata.com/cluster:
+                      {{ template "crunchy-postgres.fullname" . }}
+                    postgres-operator.crunchydata.com/instance-set: {{ .Values.instances.name }}-ha
+  users:
+    - name: {{ template "crunchy-postgres.fullname" . }}
+      databases:
+        - {{ template "crunchy-postgres.fullname" . }}
+      options: "CREATEROLE"
+    - name: postgres
+      databases:
+        - {{ template "crunchy-postgres.fullname" . }}
+  backups:
+    pgbackrest:
+      image: {{ .Values.pgbackrest.image }}
+      global:
+        # hardcoding repo1 until we solution allowing multiple repos
+        repo1-retention-full: {{ .Values.pgbackrest.retention | quote }}
+        retention-full-type: {{ .Values.pgbackrest.retentionFullType }}
```

```suggestion
        repo1-retention-full-type: {{ .Values.pgbackrest.retentionFullType }}
```
This needs the repo name prepended to the field (and will need to be dynamically generated when we figure out how to dynamically populate the list of repos).

---

Oh nice, thanks!